README.md

```diff
@@ -49,10 +49,19 @@ esxcli software vib install -v /tmp/<vib_name>.vib
 Make sure you provide the **absolute path** to the `.vib` file or the install will fail.
 
 ### On Docker Host (VM)
+#### Managed Plugin
+Please make sure to uninstall older releases of DEB/RPM if installed using following commands.
+```
+-sudo dpkg -P docker-volume-vsphere # Ubuntu or deb based distros
```

Comment:
Why "-P" and not "-r"? As per documentation "-P" removes conf files as well. Managed plugin uses same file.